### PR TITLE
[Docs] Added Queries and Metrics Pages

### PR DESCRIPTION
**Pull request by presstartgames**
^^

### CODE REVIEW

**Comment by presstartgames** [comment]
@mtrias, care to comment?

File: content/how-to-use-vividcortex/metrics.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Metrics"
+maintitle: "Metrics"
```
Comment (mtrias) [inline content/how-to-use-vividcortex/metrics.md]: not sure but... should this be _Metrics Dashboard_ ?

File: content/how-to-use-vividcortex/metrics.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+docs: "howto"
+---
+
+The Metrics page shows a list of **all** metrics, and allows you to filter by the metric template, host, and timeframe when the metric was first seen / last active.
```
Comment (mtrias) [inline content/how-to-use-vividcortex/metrics.md]: also not sure: does `template` have a meaning for the user? should me metric _name_ instead?

File: content/how-to-use-vividcortex/metrics.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+**Note**: The Metrics page was designed to show all metrics by default, regardless of values in the host selector or timepicker. Therefore, if you wish to filter by the host selector or timepicker, you should check the 'active' checkbox.
```
Comment (mtrias) [inline content/how-to-use-vividcortex/metrics.md]: isn't `timepicker` sort of an internal name? Checking [here](https://docs.vividcortex.com/how-to-use-vividcortex/time-selector/) they seem to use _Time Selector_ instead

File: content/how-to-use-vividcortex/metrics.md (hunk 1 above)
Comment (presstartgames) [inline content/how-to-use-vividcortex/metrics.md]: That works.

File: content/how-to-use-vividcortex/metrics.md (hunk 2 above)
Comment (presstartgames) [inline content/how-to-use-vividcortex/metrics.md]: Sounds better. I'll make the change.

File: content/how-to-use-vividcortex/metrics.md (hunk 3 above)
Comment (presstartgames) [inline content/how-to-use-vividcortex/metrics.md]: Update made.

**Comment by presstartgames** [comment]
I've made similar updates to the Queries page, since they are very similar.

**Comment by mtrias** [comment]
Wow! this is great! :beer:

**Comment by presstartgames** [comment]
Thanks! :beers:

**Pull request merged**